core/src/test/java/uk/gov/gchq/koryphe/impl/function/MapToTupleTest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.gchq.koryphe.impl.function;
```

Comment:
Could you add a Copyright header with 2020 as the date please

Reply:
Ok, done